src/env-config.js

```diff
@@ -22,6 +22,11 @@ const envConfig = {
       && ['false', 'disable'].includes(process.env.VUE_APP_WEBTRIT_NAVIGATION_DRAWER_MINIMIZING.toLowerCase())
   ),
 
+  webtritRecordingDisablePlayButton: !(
+    process.env.VUE_APP_RECORDING_DISABLE_PLAY_BUTTON
+      && ['false', 'disable'].includes(process.env.VUE_APP_RECORDING_DISABLE_PLAY_BUTTON.toLowerCase())
```

Comment:
For the new environment variables, we can define allowable values as desired. In this case, we can specifically handle only `true` and `false`. For the current scenario, the string `false` should be interpreted as a boolean `false`, while any other string should be considered as `true`.